Makefile

```diff
@@ -41,7 +58,11 @@ docker/weave:
 	curl -L git.io/weave -o docker/weave
 	chmod u+x docker/weave
 
-$(SCOPE_EXPORT): $(SCOPE_EXE) $(DOCKER_DISTRIB) docker/weave $(RUNSVINIT) docker/Dockerfile docker/demo.json docker/run-app docker/run-probe docker/entrypoint.sh
+docker/ebpf.tgz: Makefile
+	$(SUDO) docker pull $(EBPF_IMAGE)
+	CONTAINER_ID=$(shell $(SUDO) docker run -d $(EBPF_IMAGE) /bin/false 2>/dev/null || true); $(SUDO) docker export -o docker/ebpf.tgz $${CONTAINER_ID}
```

Comment:
>  But it is useful during development to generate a new image/artifact automatically for each branch / after each PR on tcptracer-bpf.

Right, but as you said, the same can be done with artifacts (CI can do that automatically)

> Who should review tcptracer-bpf?

I would say @brb / @awh (if they have the time) and myself. I have just created https://github.com/weaveworks/tcptracer-bpf , please create a PR there.